service/src/main/java/uk/gov/hmcts/reform/fpl/model/HearingBooking.java

```diff
@@ -60,4 +64,20 @@ public String toLabel(String dateFormat) {
         String label = OTHER == type ? capitalize(typeDetails) : type.getLabel();
         return format("%s hearing, %s", label, formatLocalDateTimeBaseUsingFormat(startDate, dateFormat));
     }
+
```

Comment:
Might need the @JsonIgnore annotation here I think.

Reply:
@joez-f what do you think? I think because it's not a getter, it doesn't need jsonIgnore.

Reply:
Yea don't need it, seems to work fine without 👍